src/components/SaveButton/SaveButton.js

```diff
@@ -1,40 +1,34 @@
-import React from "react";
-import "./SaveButton.css";
+import React, { useState } from "react";
+import styles from "./SaveButton.scss";
 
 const { ipcRenderer } = require("electron");
 
-export default class MyNavbar extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { value: "" };
+const NavBar = () => {
+  const [fileName, setFileName] = useState("");
 
-    this.handleChange = this.handleChange.bind(this);
-    this.handleSubmit = this.handleSubmit.bind(this);
-  }
+  const handleChange = event => {
+    setFileName(event.target.value);
+  };
 
-  handleChange(event) {
-    this.setState({ value: event.target.value });
-  }
-
-  handleSubmit(event) {
-    ipcRenderer.send("print:value", this.state.value);
+  const handleSubmit = event => {
+    ipcRenderer.send("print:value", fileName);
     event.preventDefault();
-  }
+  };
+
+  return (
+    <div className={styles.SaveButton}>
+      <form onSubmit={handleSubmit.bind(this)}>
```

Comment:
binding is an old way to handle events, normally you can do:
```javascript
onSumbit( e => handleSubmit(e))
```
or even e.target.value so later on in the function instead of writing 
```javascript
const handleChange = event => {
    setFileName(event.target.value);
  };
```
could be just 
```javascript
const handleChange = value => {
    setFileName(value);
  };`
```

Reply:
I went with passing whole event object